client/state/posts/actions.js

```diff
@@ -72,6 +72,8 @@ import {
 import { setEditorLastDraft, resetEditorLastDraft } from 'state/ui/editor/last-draft/actions';
 import { getSelectedSiteId } from 'state/ui/selectors';
 
+import 'state/posts/init';
```

Comment:
I wonder if we should add it to the [revisions actions](https://github.com/Automattic/wp-calypso/blob/update/modularize-posts-state/client/state/posts/revisions/actions.js#L12) and all other inner-defined actions too? Just in case someone tries to use them directly.

Reply:
Yes, we definitely should! Good catch, @tyxla!

Reply:
👍 I can see you've got it, but I've just pushed one last `init` include at the post revision author actions. Feel free to delete that commit if you think it doesn't make sense 😉

Reply:
Good catch, I missed that one!